x-pack/plugins/security/server/authentication/authenticator.ts

```diff
@@ -413,13 +418,19 @@ export class Authenticator {
           existingSessionValue,
         });
 
+        // FIXME: Clean session storage client side!!!
+        if (existingSessionValueRaw instanceof SessionExpiredError) {
+          authenticationResult.options.redirectURL += '&msg=SESSION_EXPIRED';
```

Comment:
This is currently a hack as the `options` property i private. But the `authenticationResult` object currently doesn't have a way to edit the `redirectURL` once constructed